cypress/integration/invoices/invoices.manuallyCreateLineFromPol.spec.js

```diff
@@ -0,0 +1,66 @@
+import TopMenu from '../../support/fragments/topMenu';
+import NewInvoice from '../../support/fragments/invoices/newInvoice';
+import NewInvoiceLine from '../../support/fragments/invoices/newInvoiceLine';
+import Invoices from '../../support/fragments/invoices/invoices';
+import testType from '../../support/dictionary/testTypes';
+import VendorAddress from '../../support/fragments/invoices/vendorAddress';
+import newOrder from '../../support/fragments/orders/newOrder';
+import newOrderLine from '../../support/fragments/orders/newOrderLine';
+import Orders from '../../support/fragments/orders/orders';
+
+describe('ui-invoices: Invoice Line creation - based on POL', () => {
+  const invoice = { ...NewInvoice.defaultUiInvoice };
+  const vendorPrimaryAddress = { ...VendorAddress.vendorAddress };
+  const invoiceLine = { ...NewInvoiceLine.defaultUiInvoiceLine };
+  const order = { ...newOrder.defaultOrder };
+  const orderLine = { ...newOrderLine.defaultOrderLine };
+  const locationName = 'Main Library';
+  const euroCurrency = 'Euro (EUR)';
+  const euroSign = '€';
+
+  before(() => {
+    cy.getToken(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.getOrganizationApi({ query: `name=${invoice.vendorName}` })
+      .then(({ body }) => {
+        invoice.accountingCode = body.organizations[0].erpCode;
```

Comment:
- body.organizations[0] can be moved into const or you can return this value from getOrganizationApi instead of body
- steps related with body.organizations[0].id can be moved to one method( for example setOrganizationId(...requiredObjects, ogrId){requiredObjects.foreach(obj => obj = objId)}

Reply:
Const added; As for the method - I don't know how to make it 'universal' since we're assigning different parameters from response - id, address, erpCode etc. Maybe you can suggest.